frontend/src/app/features/work-packages/components/wp-table/context-menu-helper/wp-context-menu-helper.service.ts

```diff
@@ -212,6 +218,11 @@ export class WorkPackageContextMenuHelperService {
         text: I18n.t('js.relation_buttons.add_follower'),
         link: 'addRelation',
       });
+      allowedActions.push({
+        key: 'relations',
+        text: I18n.t('js.relation_buttons.show_relations'),
+        link: 'relations',
+      });
```

Comment:
Same as above, this should not only be visible when the user has the permission to add relations, but always when the Gantt is visible.